cfgov/v1/wagtail_hooks.py

```diff
@@ -163,3 +168,84 @@ class MegaMenuModelAdmin(ModelAdmin):
 def clear_mega_menu_cache(sender, instance, **kwargs):
     from django.core.cache import caches
     caches['default_fragment_cache'].delete('mega_menu')
+
+
+def get_resource_tags():
+    tag_list = []
+
+    for resource in Resource.objects.all():
+        for tag in resource.tags.all():
+            tuple = (tag.slug, tag.name)
+            if tuple not in tag_list:
+                tag_list.append(tuple)
+
+    return sorted(tag_list, key=lambda tup: tup[0])
+
+
+class ResourceTagsFilter(admin.SimpleListFilter):
```

Comment:
Nice-to-have: unit tests for this class. There are some complex Django unit tests that might be useful [here](https://github.com/django/django/blob/1.11.15/tests/admin_filters/tests.py#L251).

Reply:
Working on this. I've added some tests for the `get_resource_tags` function above here.

Reply:
Tests for ResourceTagsFIlter have now been added!

Reply:
The `ResourceTagsFilter` tests were based on Wagtail's tests here: https://github.com/wagtail/wagtail/blob/master/wagtail/contrib/modeladmin/tests/test_simple_modeladmin.py#L53-L63